docs/vast-vpaid.html

```diff
@@ -37,20 +34,13 @@
         cloud_name: 'demo',
         ads: {
           adTagUrl: adTagUrl,
+          debug: true
```

Comment:
should not we remove this now?

Reply:
I'm now making this configurable, instead of always `true`.
I don't mind it will be `true` in the example page